### ISSUE TITLE
The video file generated by FrameRecoder is indicated broken by windows media player

### ISSUE THREAD

**Issue by freshcoder39** [bytedeco/javacv#1202]
Hi, the version of my javacv platform is 1.3.3, I would like to use it to output the video from the laptop camera. However, the output file whose format is mp4  is indicated that is broken.

I have tested the file with various media player, the results were the same.
The following is some parts of my code

FrameRecorder recorder = FrameRecorder.createDefault(outputFile, width, height);
        recorder.setVideoCodec(avcodec.AV_CODEC_ID_H264); 
        recorder.setPixelFormat(avutil.AV_PIX_FMT_YUV420P);
        recorder.setFormat("mp4");
        recorder.setFrameRate(frameRate);

        frame.dispose();
        recorder.stop();
        recorder.release();
        grabber.stop();

I would like to know that is somebody met the same issue?

**Comment by saudet** [comment]
Please try again with JavaCV 1.5.

**Comment by freshcoder39** [comment]
Sure, I will try the Javacv 1.5 at first

**Comment by freshcoder39** [comment]
> Please try again with JavaCV 1.5.

Hi, I ran the demo with version latest 1.5
https://github.com/bytedeco/javacv/blob/master/samples/WebcamAndMicrophoneCapture.java

In order to get the video from the camera integrated on the laptop and display it on the browser
I set the output video to the d:/apache/htdocs, so that I can access it on the browser.
 The video produced but it cannot play on the media player.

**I set the device number to 0**
I set the **recorder.setFormat("mp4")**, for the output format, is mp4 which is supported by the browser
What's more, **I tried the recorder.setFormat("flv")** with the output format mp4.
However, all the videos produced with the above parameters are broken. 

I would like to ask which parameter effect the produce with the video format, besides the recorder.setFormat().
